econml/utilities.py

```diff
@@ -570,7 +570,8 @@ def get_input_columns(X, prefix="X"):
         return None
     if np.ndim(X) == 0:
         raise ValueError(
-            "Expected array-like object for imput with prefix {prefix} but got '{X}' object instead.".format(prefix, X))
+            "Expected array-like object for input \
+            with prefix {prefix} but got '{X}' object instead.".format(prefix, X))
```

Comment:
Use a format literal string:

```suggestion
            f"Expected array-like object for imput with prefix {prefix} but got '{X}' object instead.")
```